incubator/airflow/values.yaml

```diff
@@ -0,0 +1,297 @@
+    ## header like the requested URL (no special configuration neeed)
+    annotations:
+      ##
+      ## Example for Traefik:
+      # traefik.frontend.rule.type: PathPrefix
+      # kubernetes.io/ingress.class: traefik
+  ##
+  ## Configure the flower endpoind
+  flower:
+    ##
+    ## if flower is '/airflow/flower':
+    ##  - Flower UI is at 'http://mycompany.com/airflow/flower'
+    ## NOTE: you need to have a reverse proxy/load balancer able to do URL rewrite in order to have
+    ## flower mounted on other path than root. Flower only does half the job in url prefixing: it
+    ## only generates the right URL/relative paths in the **returned HTML files**, but expects the
+    ## request to have been be at the root.
+    ## That's why we need a reverse proxy/load balancer that is able to strip the path
+    ## NOTE: do NOT keep trailing slash. For root configuration, set and empty string
+    path: ""
+    ##
+    ## Configure the liveness path. Keep to "/" for Flower >= jan 2018.
+    ## For previous version, enter the same path than in the 'path' key
+    ## NOTE: keep the trailing slash.
+    liveness_path: /
+    ##
+    ## hostname for flower
+    host: ""
+    ##
+    ## Annotation for the Flower endpoint
+    ##
+    ## ==== SKIP THE FOLLOWING BLOCK IF YOU HAVE FLOWER > JANUARY 2018 =============================
+    ## Please note their is a small difference between the way Airflow Web server and Flower handles
+    ## URL prefixes in HTTP requests:
+    ## Flower wants HTTP header to behave like there was no URL prefix, and but still generates
+    ## the right URL in html pages thanks to its `--url-prefix` parameter
+    ##
+    ##    Extracted from the Flower documentation:
+    ##    (https://github.com/mher/flower/blob/master/docs/config.rst#url_prefix)
+    ##
+    ##        To access Flower on http://example.com/flower run it with:
+    ##            flower --url-prefix=/flower
+    ##
+    ##        Use the following nginx configuration:
+    ##            server {
+    ##              listen 80;
+    ##              server_name example.com;
+    ##
+    ##              location /flower/ {
+    ##                rewrite ^/flower/(.*)$ /$1 break;
+    ##                proxy_pass http://example.com:5555;
+    ##                proxy_set_header Host $host;
+    ##              }
+    ##            }
+    ## ==== IF YOU HAVE FLOWER > JANUARY 2018, NO MORE NEED TO STRIP THE PREFIX ====================
+    annotations:
+      ##
+      ## NOTE: it is important here to have your reverse proxy strip the path/rewrite the URL
+      ## Example for Traefik:
+      # traefik.frontend.rule.type: PathPrefix       ## Flower >= Jan 2018
+      # traefik.frontend.rule.type: PathPrefixStrip  ## Flower < Jan 2018
+      # kubernetes.io/ingress.class: traefik
+
+
+##
+## Storage configuration for DAGs
+persistence:
+  ##
+  ## enable persistance storage
+  enabled: false
+  ##
+  ## Existing claim to use
+  existingclaim: nil
```

Comment:
`existingclaim` needs the C to b capitalized

Reply:
indeed, thanks!